ci/jobs/synthetic-data-ingestion-dev.yml

```diff
@@ -0,0 +1,18 @@
+jobs:
+  - name: synthetic-data-ingestion-dev
+    plan:
+      - put: meta
+        resource: meta-development
+      - get: aws-analytical-dataset-generation
```

Comment:
It would be good if we got into the habit of using `aggregate` on our repository gets, if that's OK? I will communicate an example via Slack.